dateparser/custom_language_detection/langdetect.py

```diff
@@ -0,0 +1,38 @@
+from langdetect.detector_factory import DetectorFactory, PROFILES_DIRECTORY
+import langdetect
+
+
+# The below _Factory is set to prevent setting global state of the library
+# but still get consistent results.
+# Refer - https://github.com/Mimino666/langdetect
+
+class _Factory:
+    data = None
+
+
+def _init_factory():
+    if _Factory.data is None:
+        _Factory.data = DetectorFactory()
+        _Factory.data.load_profile(PROFILES_DIRECTORY)
+        _Factory.data.seed = 0
+
+
+def _get_language_probablities(text):
+    _init_factory()
+    detector = _Factory.data.create()
+    detector.append(text)
+    return detector.get_probabilities()
+
+
+def detect_languages(text, confidence_threshold):
+    language_codes = []
+    try:
+        parser_data = _get_language_probablities(text)
+        for langauge_candidate in parser_data:
+            if langauge_candidate.prob > confidence_threshold:
+                language_codes.append(langauge_candidate.lang)
+    except langdetect.lang_detect_exception.LangDetectException:
+        # This exception can be produced with empty strings or inputs without letters like `10-10-2021`.
+        # As this could be really common, we ignore them.
+        pass
+    return language_codes
```

Comment:
To cover these two lines (34 and 37) probably you can add a test like

`test_lang_detect_doesnt_raise_error`

For example after:

`test_custom_language_detect_lang_detect`

And using lang_detect, introducing a string that would produce an error and check that this error is not produced.

Reply:
Thanks, @noviluni, All tests are passing even coverage 😃😃